examples/flutter_gallery/lib/demo/material/elevation_demo.dart

```diff
@@ -0,0 +1,63 @@
+import 'package:flutter/material.dart';
+
+class ElevationDemo extends StatefulWidget {
+  static const String routeName = '/material/elevation';
+
+  @override
+  State<StatefulWidget> createState() => _ElevationDemoState();
+}
+
+class _ElevationDemoState extends State<ElevationDemo> {
+  bool _shouldShowElevation = true;
+
+  List<Widget> buildCards() {
+    final List<Widget> cards = <Widget>[];
+    const List<double> elevations = <double>[
+      0.0,
+      1.0,
+      2.0,
+      3.0,
+      4.0,
+      5.0,
+      8.0,
+      16.0,
+      24.0,
+    ];
+
+    for (double i in elevations) {
+      cards.add(new Center(
+        child: new Card(
```

Comment:
Cards already create a Semantics for which `container: true` by default, so this should be good vis. a11y.